pkg/testworkflows/testworkflowexecutor/executor.go

```diff
@@ -465,6 +465,7 @@ func (e *executor) initialize(ctx context.Context, workflow *testworkflowsv1.Tes
 		DisableWebhooks:           request.DisableWebhooks,
 		Tags:                      map[string]string{},
 		RunningContext:            request.RunningContext,
+		Config:                    request.Config,
```

Comment:
* It may contain sensitive data
* The data in the configuration may be huge in size